documentation/slate/source/includes/authentication.md

```diff
@@ -36,3 +36,7 @@ or
 
 `curl https://3b036afe-0110-4202-b9ed-99718476c2e0@api.navitia.io/v1/coverage`
 
+<aside class="notice">
```

Comment:
I would have change the begining of this chapter. For example
_For example, in a [Curl](https://en.wikipedia.org/wiki/CURL) way, you can request either :_

_For example, in a [Curl](https://en.wikipedia.org/wiki/CURL) way, you can request either (using the fake sandbox token) :_